resources/views/components/threads/thread.blade.php

```diff
@@ -57,6 +57,14 @@ class="prose prose-lio max-w-none p-6 break-words"
     >
     </div>
 
+    @if($thread->updatedBy())
+        <div class="text-sm text-gray-900 p-6">
+            Last edit by
+            <a href="{{ route('profile', $thread->updatedBy()->username()) }}" class="text-lio-500 border-b-2 pb-0.5 border-lio-100 hover:text-lio-600">{{ $thread->updatedBy()->name() }}</a>
```

Comment:
```suggestion
            <a href="{{ route('profile', $thread->updatedBy()->username()) }}" class="text-lio-500 border-b-2 pb-0.5 border-lio-100 hover:text-lio-600">{{ '@'.$thread->updatedBy()->name() }}</a>
```

Reply:
This will result  **Last edit by @Sanjit Singh on 12 July  2021** which looks bit off to me. 

@driesvints thoughts?